### PR TITLE
Add support of ARM64

### PR DESCRIPTION
**Pull request by amulobaev**

### CODE REVIEW

File: libexec/tfenv-install [hunk 1]
```diff
@@ -81,7 +81,16 @@ if [ -f "${dst_path}/terraform" ]; then
   exit 0;
 fi;
 
-TFENV_ARCH="${TFENV_ARCH:-amd64}";
+# Add support of Linux ARM64
+case "$(uname -m)" in
+  aarch64*)
```
Comment (gchenuet) [inline libexec/tfenv-install:86]: Please add support for Apple Silicon CPU:

`  aarch64* | arm64*)`

File: libexec/tfenv-install [hunk 2]
```diff
@@ -81,7 +81,16 @@ if [ -f "${dst_path}/terraform" ]; then
   exit 0;
 fi;
 
-TFENV_ARCH="${TFENV_ARCH:-amd64}";
+# Add support of Linux ARM64
```
Comment (gchenuet) [inline libexec/tfenv-install:84]: `# Add support of ARM64 for Linux & Apple Silicon`

**Review by gchenuet — CHANGES_REQUESTED** [review]
Adding Apple Silicon arm64 support.

Thanks for your work !

**Comment by amulobaev** [comment]
> Adding Apple Silicon arm64 support.
> 
> Thanks for your work!

Thank you

**Review by Zordrak — APPROVED** [review]

**Comment by danielcompton** [comment]
@Zordrak would you be able to tag a new release to include this when you get a chance? ref: https://github.com/tfutils/tfenv/issues/306

**Comment by nathanhruby** [comment]
Note also that golang/go#42700 affects the amd version and sometimes tf barfs as a result

**Pull request merged**